packages/pglite/tests/targets/deno/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "pglite-deno-tests",
+  "type": "module",
```

Comment:
nit: add `"private": true` so it gets ignored by things like changesets (even if it's not part of the workspace)